include/sys/zfs_context.h

```diff
@@ -273,6 +273,7 @@ typedef struct kmutex {
 } kmutex_t;
 
 #define	MUTEX_DEFAULT	0
+#define	MUTEX_FSTRANS	MUTEX_DEFAULT
```

Comment:
Dear @maxximino and @behlendorf   THANK YOU for spelling it out for me -- very much appreciated!  I will try to get to ZFS patching later today/tomorrow (I guess I should just take this patch alone for clarity) and report back in a week or earlier if stall occurs again ;-)